### PR TITLE
Added support for the Auvidea J100 / RPi 2.x Camera

### PR DESCRIPTION
**Pull request by jallum**
Included are the necessary DTB, machine configuration and kernel
patches necessary to support the Auvidea J100 carrier board, as
well as the IMX219 (Rasperry PI 2.x) camera module.

We are able to boot off of this image, flash a Jetson (the J100 has an
on-board SD card slot) and see our devices.

### CODE REVIEW

File: layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_4.4.bbappend [hunk 1]
```diff
@@ -41,3 +42,9 @@ RESIN_CONFIGS_append = " serial"
 RESIN_CONFIGS[serial] = " \
 		CONFIG_USB_SERIAL_GENERIC=y \
 		"
+
+RESIN_CONFIGS_append = " imx219"
```
Comment (floion) [inline layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_4.4.bbappend]: Please add these configs to linux-tegra_%.bbappend instead.
And when you do, please add them only for your machine. For example by doing: 

> RESIN_CONFIGS_append_j100-tx2 = " imx219"

**Comment by floion** [comment]
Hi @jallum thanks for the contribution. I left a comment for your PR.
Also, please follow the contribution guideline: https://github.com/balena-os/balena-jetson-tx2#contributing (you are missing a "Changelog-entry" in your git commit log)

**Comment by floion** [comment]
@resin-jenkins ok to test

**Comment by floion** [comment]
@jallum can you restructure this PR so that each file change would go in it's own commit? You can use this pull request https://github.com/balena-os/balena-jetson-tx2/pull/130 as an example

**Comment by floion** [comment]
This PR also needs rebasing when you redo it

**Comment by Jaeyoung-Lim** [comment]
Anyplans on also supporting the J120 carrier board?

**Comment by floion** [comment]
Hi, we don't have yet any customers using J120. But if are using that board, you can PR the required changes for it, using an existing PR as a model. Let me know if you want to PR it and I'll show you a few examples

**Comment by floion** [comment]
This repo should not be used anymore for newer boards so let's close this PR

**Pull request closed without merging**